pkg/storage/secret/metadata/decrypt_store_test.go

```diff
@@ -59,7 +59,7 @@ func TestIntegrationDecrypt(t *testing.T) {
 
 		exposed, err := decryptSvc.Decrypt(authCtx, "default", "non-existent-value")
 		require.Error(t, err)
-		require.Contains(t, err.Error(), "could not get secure value")
+		require.Equal(t, err.Error(), "not found")
```

Comment:
Can also remove line 61

```suggestion
		require.ErrorIs(t, err, contracts.ErrDecryptNotFound)
```